octobot_trading/channels/orders.py

```diff
@@ -104,20 +105,34 @@ async def handle_post_open_order_update(self, symbol, orders, has_new_order):
     async def update_order_from_exchange(self, order,
                                          should_notify=False,
                                          wait_for_refresh=False,
-                                         force_job_execution=False):
+                                         force_job_execution=False,
+                                         should_create_try_to_create_producer=True):
```

Comment:
I think naming this param something like `create_order_producer_if_missing` would be better

Reply:
Yes good idea :+1: